src/app/frontend/podexec/afterrender_directive.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Returns directive that runs code after a template render.
+ *
+ * @return {!angular.Directive}
+ * @ngInject
+ */
+export default function afterRenderDirective($timeout) {
+  var def = {
+    restrict: 'A',
+    terminal: true,
+    transclude: false,
+    /** @param {!AfterRender.attrs} attrs */
+    link: function (scope, element, attrs) {
```

Comment:
As discussed offline, this is my suggestion that should work.

1. Rename this from generic 'after-render' to 'terminal-foo' or something.
2. Attach terminal to the `element` variable you have here, e.g., by taking the lines from podexec_controller.js:
```js
   // TODO: Is there a better way to get the element to attach the terminal to?
    this.term.open(this.document_.getElementById('kd-podexec-terminal'));
```

To:

```js
   this.term.open(element);
```